### PR TITLE
Using go graphviz library to print dot file

### PR DESCRIPTION
**Pull request by pmahindrakar-oss**
Signed-off-by: Prafulla Mahindrakar <prafulla.mahindrakar@gmail.com>

# TL;DR
Changed to using the go graphviz library

Following are some of the renderd graph results

https://graph.flyte.org/#strict%20digraph%20%20%7B%0A%09n1_n0-%3En1_end_node%3B%0A%09n1_n1-%3En1_end_node%3B%0A%09n1_n0-%3En1_n1%3B%0A%09n1_start_node-%3En1_n0%3B%0A%09start_node-%3Enode_t1_parent%3B%0A%09n1_start_node-%3Eend_node%3B%0A%09node_t1_parent-%3Eend_node%3B%0A%09node_t1_parent-%3En1_start_node%3B%0A%09subgraph%20cluster_n1%20%7B%0A%09n1_end_node%20%5B%20color%3Dred%2C%20label%3Dend%2C%20shape%3Ddoublecircle%20%5D%3B%0A%09n1_n0%20%5B%20label%3D%22t1%20%5Bpython-task%5D%22%2C%20shape%3Dbox%20%5D%3B%0A%09n1_n1%20%5B%20label%3D%22t1%20%5Bpython-task%5D%22%2C%20shape%3Dbox%20%5D%3B%0A%09n1_start_node%20%5B%20color%3Dgreen%2C%20label%3Dstart%2C%20shape%3Ddoublecircle%20%5D%3B%0A%0A%7D%0A%3B%0A%09end_node%20%5B%20color%3Dred%2C%20label%3Dend%2C%20shape%3Ddoublecircle%20%5D%3B%0A%09node_t1_parent%20%5B%20label%3D%22t1%20%5Bpython-task%5D%22%2C%20shape%3Dbox%20%5D%3B%0A%09start_node%20%5B%20color%3Dgreen%2C%20label%3Dstart%2C%20shape%3Ddoublecircle%20%5D%3B%0A%0A%7D

https://graph.flyte.org/#strict%20digraph%20%20%7B%0A%09n0-%3Eend_node%3B%0A%09n1-%3Eend_node%3B%0A%09n0-%3En1%3B%0A%09start_node-%3En0%3B%0A%09start_node-%3En1%3B%0A%09end_node%20%5B%20color%3Dred%2C%20label%3Dend%2C%20shape%3Ddoublecircle%20%5D%3B%0A%09n0%20%5B%20label%3D%22t1%20%5Bpython-task%5D%22%2C%20shape%3Dbox%20%5D%3B%0A%09n1%20%5B%20label%3D%22t2%20%5Bpython-task%5D%22%2C%20shape%3Dbox%20%5D%3B%0A%09start_node%20%5B%20color%3Dgreen%2C%20label%3Dstart%2C%20shape%3Ddoublecircle%20%5D%3B%0A%0A%7D

https://graph.flyte.org/#strict%20digraph%20%20%7B%0A%09n0-%3En0_n0%5B%20label%3D%22(.my_input%20GTE%200.1)%20AND%20(.my_input%20LTE%201)%22%20%5D%3B%0A%09n0-%3En0_n1%5B%20label%3D%22orElse%22%20%5D%3B%0A%09n0-%3Eend_node%3B%0A%09start_node-%3En0%3B%0A%09start_node-%3En0_n0%3B%0A%09start_node-%3En0_n1%3B%0A%09subgraph%20cluster_fractions%20%7B%0A%09n0%20%5B%20label%3Dfractions%2C%20shape%3Ddiamond%20%5D%3B%0A%09n0_n0%20%5B%20label%3D%22double%20%5Bpython-task%5D%22%2C%20shape%3Dbox%20%5D%3B%0A%09n0_n1%20%5B%20label%3D%22square%20%5Bpython-task%5D%22%2C%20shape%3Dbox%20%5D%3B%0A%0A%7D%0A%3B%0A%09end_node%20%5B%20color%3Dred%2C%20label%3Dend%2C%20shape%3Ddoublecircle%20%5D%3B%0A%09start_node%20%5B%20color%3Dgreen%2C%20label%3Dstart%2C%20shape%3Ddoublecircle%20%5D%3B%0A%0A%7D

## Type
 - [ ] Bug Fix
 - [X] Feature
 - [ ] Plugin

## Are all requirements met?
Pending 
* using correct lhead and ltail to fix the arrows in subgraphs
* Unit tests
* Adding the option to output format 

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

File: cmd/get/workflow.go [hunk 1]
```diff
@@ -108,14 +108,14 @@ func getWorkflowFunc(ctx context.Context, args []string, cmdCtx cmdCore.CommandC
 			return err
 		}
 		if len(workflows) > 0 && workflowconfig.DefaultConfig.Visualize != "" {
-			f, err := workflowconfig.DefaultConfig.GraphvizFormat()
-			if err != nil {
-				return err
-			}
+			//f, err := workflowconfig.DefaultConfig.GraphvizFormat()
```
Comment (kumare3) [inline cmd/get/workflow.go:111]: delete

File: cmd/get/workflow.go [hunk 2]
```diff
@@ -108,14 +108,14 @@ func getWorkflowFunc(ctx context.Context, args []string, cmdCtx cmdCore.CommandC
...
+			//if err != nil {
+			//	return err
+			//}
 			if workflowconfig.DefaultConfig.OutputFile == "" {
```
Comment (kumare3) [inline cmd/get/workflow.go:115]: delete only use --output "dot"

**Comment by kumare3** [comment]
this looks good to me

**Comment by pmahindrakar-oss** [comment]
@kumare3  I am merging this step and taking your feedback in the followup commit which i am working on
